### DISCUSSION TITLE
Alternative light sources suggestion

### DISCUSSION THREAD

**Discussion by monte-monte** [stevenbennett/DIYson#17] [Ideas]
First of all, I'd like to say that I am very happy to find your project and that you keep everything so well documented and open. Your approach is really giving me an inspiration for my own work. I am working on my own lamp project and have sought cheap and readily available solution for a light module of some sort. So I bought a few types of LED spotlights on aliexpress for testing. I am driving them with custom circuitry so I'm not concerned with power supply's quality they came with. 
I understand that this may be pretty big step away from your design and vision, but for someone who will try to repeat your project for themselves it could be easier to source mentioned chinese lamp modules.
The ones I've bought: https://www.aliexpress.com/item/1005004043668699.html, https://www.aliexpress.com/item/1005002848822524.html, https://www.aliexpress.com/item/1005004267786865.html
They are bulkier then your assembly, but I think some of them can be housed in some clever 3dprinted mount to better suit the overall aesthetics.

**Comment by stevenbennett** [comment]
@monte-monte Very cool, definitely a nice head start to find a compact little package with an LED, electronics, and a light diffuser. Curious to see where your project takes you :)
